awx/ui/src/screens/Template/WorkflowJobTemplateVisualizer/Modals/NodeModals/NodeTypeStep/useNodeTypeStep.js

```diff
@@ -61,7 +61,7 @@ function getStep(
       (nodeTypeField.value !== 'workflow_approval_template' &&
         nodeResourceField.value === null) ||
       (nodeTypeField.value === 'workflow_approval_template' &&
-        approvalNameField.value === undefined) ||
+        approvalNameField.value === '') ||
```

Comment:
@AlexSCorey That it was previously `undefined` makes me think there's probably a scenario where the value _is_ `undefined` (This can happen on the survey form, if I recall). 

I'm wondering if we should add the empty string check to the existing conditional rather than replace it.
```suggestion
        approvallNameField.value === 'undefined' || approvalNameField.value === '') ||
```

Reply:
discussed w/ @AlexSCorey - we're reasonably sure this value can never be `undefined`, so leaving as is.